panel.js

```diff
@@ -48,13 +48,10 @@ var PanelIndicator = GObject.registerClass(
       this.settings = settings;
       this.extWm = extWm;
 
-      let tileIconOff = Gio.icon_new_for_string(
-        `${Me.path}/icons/panel/focus-windows-symbolic.svg`
-      );
-      let tileIconOn = Gio.icon_new_for_string(`${Me.path}/icons/panel/view-dual-symbolic.svg`);
-      let workspaceIconOff = Gio.icon_new_for_string(
-        `${Me.path}/icons/panel/window-duplicate-symbolic.svg`
-      );
+      const panelIconsPath = `${Me.path}/resources/icons/hicolor/scalable/panel`
```

Comment:
there's probably a better way to do this like 
```js
const tileIconOff = somehowGetGIconFromName('focus-windows-symbolic');
```

Reply:
Originally, aspiring to import other icons liked from community themes and packaged it with the extension. For consistency, I would agree to use the system symbolic (so you can usw the names) versus the one that's implemented today.

Reply:
One isue, is that we lost the ability to update the icon when the tiling mode status changes:

- Workspace floating mode
- Tiling is ON or OFF